syntax = "proto2";

package opentxs.proto;
option java_package = "org.opentransactions.proto";
option java_outer_classname = "OTStorageItemHash";
option optimize_for = LITE_RUNTIME;

import public "StorageEnums.proto";

message StorageItemHash {
    optional uint32 version = 1;
    optional string itemID = 2;
    optional string hash = 3;
    optional string alias = 4;
    optional StorageHashType type = 5;
}
